StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/DocumentationResources.ru-RU.resx

```diff
@@ -139,16 +139,16 @@
     <value />
   </data>
   <data name="StartingTextGets" xml:space="preserve">
-    <value>Получает</value>
+    <value>Возвращает</value>
   </data>
   <data name="StartingTextGetsOrSets" xml:space="preserve">
-    <value>Получает или задает</value>
+    <value>Возвращает или задает</value>
   </data>
   <data name="StartingTextGetsOrSetsWhether" xml:space="preserve">
-    <value>Получает или задает значение, показывающее,</value>
```

Comment:
⚠️ This change is _partially_ consistent with an example page I found:
https://docs.microsoft.com/ru-ru/dotnet/api/system.windows.forms.control.isaccessible?view=netframework-4.8

StyleCop Analyzers currently uses `показывающее` (before and after this pull request), where the example uses `указывающее`. I have no idea which one is correct.

Reply:
I wasn't focussed on it, so I can't anything about.

Reply:
`показывающее` is `showing`
`указывающее` is `pointing` / `pointing out`

But I haven't looked looked at the context, so it might be slightly incorrect